### PR TITLE
feat: add duration of the appearance Badge

### PR DESCRIPTION
**Pull request by sawel24**
## Connection with issue(s)

Resolve issue https://github.com/yako-dev/flutter_badges/issues/67

Adds new variable a duration of the appearance Badge

## Testing and Review Notes
You need to create a Badge widget, set the appearanceDration, and check how long the Badge will animate before full appearance
<!-- Required: --

### CODE REVIEW

**Comment by yadaniyil** [comment]
@sawel24 so you added new parameter, but used the old one? really?
image

**Review by yadaniyil — CHANGES_REQUESTED** [review]
Not working

**Comment by sawel24** [comment]
> Not working

I changed it and it should work now

File: lib/src/badge.dart [hunk 1]
```diff
@@ -94,6 +95,11 @@ class Badge extends StatefulWidget {
   /// * [animationType]
   final Duration animationDuration;
 
+  /// Сontrols the duration of the appearance Badge.
```
Comment (yadaniyil) [inline lib/src/badge.dart:98]:
```suggestion
  /// Duration of the badge appearance and disappearance animations.
  /// Set this to zero to skip the badge appearance and disappearance animations
```

**Review by yadaniyil — CHANGES_REQUESTED** [review]

**Pull request merged**